### DISCUSSION TITLE
typo in "0_how_to_work_with_onnx" notebook

### DISCUSSION THREAD

**Discussion by amdPaek** [fastmachinelearning/qonnx#93] [General]
The first cell in this notebook has a typo:

#from qonnx.basic.util import qonnx_make_model
from qonnx.util.basic import qonnx_make_model

**Comment by maltanar** [comment]
Thanks @amdPaek , this was addressed in https://github.com/fastmachinelearning/qonnx/pull/81 . 
For the future, please use the [Issues](https://github.com/fastmachinelearning/qonnx/issues) tab to report issues instead.
